### PR TITLE
BeforeSuite addMochawesome Context support and DDT improve

### PR DESCRIPTION
**Pull request by APshenkin**
BeforeSuite addMochawesome Context support
Add `xadd` for datatables to ignore this tests in run
Add `Data(table).only.Scenario` to give ability to launch only `Data` tests

### CODE REVIEW

File: lib/helper/Mochawesome.js [hunk 1]
```diff
@@ -67,6 +68,7 @@ class Mochawesome extends Helper {
   }
 
   addMochawesomeContext(context) {
+    if (currentTest = "") currentTest = { test: currentSuite.ctx.test }
```
Comment (jploskonka) [inline lib/helper/Mochawesome.js]: Hmm, shouldn't this be `currentTest ==`?  Or am I missing something? :)

Comment (APshenkin) [reply lib/helper/Mochawesome.js]: Yep will fix this)

File: lib/helper/Mochawesome.js [hunk 2]
```diff
@@ -32,6 +32,7 @@ class Mochawesome extends Helper {
 
   _beforeSuite(suite) {
     currentSuite = suite;
+    currentTest = ""
```
Comment (DavertMik) [inline lib/helper/Mochawesome.js]: `;` - in the end :)

File: lib/data/table.js [hunk 3]
```diff
@@ -19,7 +19,19 @@ class DataTable {
       tempObj.toString = () => JSON.stringify(tempObj);
       arrayCounter++;
     });
-    this.rows.push(tempObj);
+    this.rows.push({ skip: false, data: tempObj});
+  }
+
+  xadd(array) {
```
Comment (DavertMik) [inline lib/data/table.js]: where is xadd used?
probably it should be documented, right?

Comment (APshenkin) [reply lib/data/table.js]: Updated.

**Comment by DavertMik** [comment]
Thanks! Looks good to me

**Pull request merged**